doc/MANUAL.md

```diff
@@ -245,6 +243,18 @@ This option shows program's version number and exits.
 
 This option skips checking for a new version of the program.
 
+## NVD Arguments
+
+### -u {now,daily,never}, --update {now,daily,never}
+
+This option controls the frequency of updates for the CVE data from the National Vulnerability Database.  By default, the tool checks the staleness of the data with every run, and if the data is more than one day old, it gets an update from NVD.  You may also choose to update the data `now` (in which case all cached data is deleted and a full new download is done) or `never` in which case the staleness check is not done and no update is requested.  The `now` and `never` modes can be combined to produce alternative update schedules if daily is not the desired one.
+
+### -n {json,api}, --nvd {json,api}
+
+This option allows you to set the medium CVE data is updated from the National Vulnerability Database. By default, `json` is enabled, in which the data is fetched using the JSON feeds available on [this page](https://nvd.nist.gov/vuln/data-feeds). These year feeds are updated once per day.  
```

Comment:
```suggestion
This option allows you to set the medium in which CVE data is updated from the National Vulnerability Database. By default, `json` is enabled, meaning the data is fetched using the JSON feeds available on [this page](https://nvd.nist.gov/vuln/data-feeds). These year feeds are updated once per day.  
```

Reply:
```suggestion
This option selects how CVE data is downloaded from the National Vulnerability Database.  The default `json` option uses the JSON feeds available on [this page](https://nvd.nist.gov/vuln/data-feeds). These per-year feeds are updated once per day.  
```

I don't love the word which (a personal preference borne of a prof I used to work with on dense academic papers) so it stood out in the diff.  Here's a go at shortening and taking out "which" entirely.